app/models/organization/OrganizationService.scala

```diff
@@ -38,6 +40,8 @@ class OrganizationService @Inject()(organizationDAO: OrganizationDAO,
     } else Json.obj()
     for {
       usedStorageBytes <- organizationDAO.getUsedStorage(organization._id)
+      owner <- userDAO.findOwnerByOrg(organization._id)(GlobalAccessContext)
+      ownerName = s"${owner.firstName} ${owner.lastName}"
```

Comment:
```suggestion
      ownerBox <- userDAO.findOwnerByOrg(organization._id).futureBox
      ownerNameOpt = ownerBox.toOption.map(o => s"${o.firstName} ${o.lastName}")
```

There is not currently any code guaranteeing that there exists an owner. While this is kind of an invalid state, I’d add this safeguard here to avoid breaking the entire organization request in this case. I think philipp also already talked about making the ownerName property optional in the frontend.